src/components/ToggleButtons.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+
+const BUTTON_CLASSNAME = 'py-3 w-full text-xl font-normal rounded-lg';
+
+export default function ToggleButtons() {
+  const [isSignInOn, setIsSignInOn] = useState(true);
```

Comment:
Здесь лучше использовать конкретный ключ, а не булевое значение


```suggestion
  // 2 варианта 'signIn' или 'register'
  const [activeTab, setActiveTab] = useState('signIn');
```